WordPress/twentytwentyfour
# Performance improvement: Ensure styles for the block-style variations only get loaded when the related blocks are rendered

**Description**

This is a performance improvement to ensure that styles for block-styles variations only get enqueued when the related blocks and variations actually exist on the page.

**Testing Instructions**
* Details block: Ensure that the "Arrow icon" style looks as it did before.
* Post Terms block: Ensure that the "Pill" style looks as it did before.
* List block: Ensure that the "Checkmark" style looks as it did before.
